src/include/migraphx/op/nonmaxsuppression.hpp

```diff
@@ -82,7 +82,7 @@ iou_threshold (optional) : tensor(float)
 Float representing the threshold for deciding whether boxes overlap too much with respect to IOU.
 It is scalar. Value range [0, 1]. Default to 0.
 
-score_threshold (optional) : tensor(flo187Gat)
+score_threshold (optional) : tensor(float)
```

Comment:
This looks like this was blundering late at night with this mess. Thanks for cleaning it up